content/v2/tlds.markdown

```diff
@@ -0,0 +1,221 @@
+---
+title: TLDs | DNSimple API v2
+excerpt: This page documents the DNSimple TLDs API v2.
+---
+
+# TLDs API
+
+* TOC
+{:toc}
+
+
+## List TLDs {#list}
+
+    GET /tlds
+
+### Example
+
+List all TLDs.
+
+    curl -H 'Authorization: Bearer <token>' \
+         -H 'Accept: application/json' \
+         https://api.dnsimple.com/v2/tlds
+
+### Response
+
+~~~json
+{
+  "data":[
+    {"tld":"com", "tld_type":1, "whois_privacy":false, "auto_renew_only":false},
+    {"tld":"ca", "tld_type":1, "whois_privacy":false, "auto_renew_only":false}
+  ]
+}
+~~~
+
+### TLD Types
+
+Value | Type
+------|-----
+`1`   | `gTLD`
+`2`   | `ccTLD`
+`3`   | `newTLD`
+
+## Get a TLD {#get}
+
+    GET /tlds/:tld
+
+### Parameters
+
+Name | Type | Description
+-----|------|------------
+`:tld` | `string` | The TLD name
+
+### Example
+
+Get the TLD with name `"ca"`.
+
+    curl -H 'Authorization: Bearer <token>' \
+         -H 'Accept: application/json' \
+         https://api.dnsimple.com/v2/tlds/com
+
+### Response
+
+~~~json
+{
+  "data":{
+    "tld":"ca", "tld_type":1, "whois_privacy":false, "auto_renew_only":false
+  }
+}
+~~~
+
+## Get TLD Extended Attributes {#extendedAttributes}
+
+    GET /tlds/:tld/extended_attributes
+
+### Parameters
+
+Name | Type | Description
+-----|------|------------
+`:tld` | `string` | The TLD name
+
+### Example
+
+Get extended attributes for TLD with name `"ca"`.
+
+    curl -H 'Authorization: Bearer <token>' \
+         -H 'Accept: application/json' \
+         https://api.dnsimple.com/v2/tlds/ca/extended_attributes
+
+### Response
+
+~~~json
+{
+  "data"=>[
```

Comment:
This is not JSON :smile: 

FYI, just leave the response empty, I've been reusing the fixtures we created for the API client. See 13e7832380f286ded33c636e87c43a3d866006a7